pkg/kubelet/images/image_manager_test.go

```diff
@@ -241,17 +244,17 @@ func pullerTestEnv(t *testing.T, c pullerTestCase, serialized bool, maxParallelI
 	fakeRuntime = &ctest.FakeRuntime{T: t}
 	fakeRecorder := &record.FakeRecorder{}
 
-	fakeRuntime.ImageList = []Image{{ID: "present_image:latest"}}
+	fakeRuntime.ImageList = []kubecontainer.Image{{ID: "present_image:latest"}}
 	fakeRuntime.Err = c.pullerErr
 	fakeRuntime.InspectErr = c.inspectErr
 
 	fakePodPullingTimeRecorder = &mockPodPullingTimeRecorder{}
-
-	puller = NewImageManager(fakeRecorder, fakeRuntime, backOff, serialized, maxParallelImagePulls, c.qps, c.burst, fakePodPullingTimeRecorder)
+	puller = NewImageManager(fakeRecorder, fakeRuntime, backOff, serialized, maxParallelImagePulls, c.qps, c.burst, fakePodPullingTimeRecorder, nil, metav1.Duration{Duration: 24 * time.Hour})
 	return
 }
 
 func TestParallelPuller(t *testing.T) {
+	defer os.RemoveAll(tmpStateDir)
```

Comment:
it's not clear to me why we're removing this in a test that doesn't apparently use it

Reply:
This needs to be removed. It used for checkpoint saving before.